globus_sdk/nexus.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import print_function
+
+from globus_sdk.base import BaseClient
+
+
+class NexusClient(BaseClient):
+    def __init__(self, environment="default"):
+        BaseClient.__init__(self, "nexus", environment)
+
+    def get_goauth_token(self, username, password):
```

Comment:
I was actually thinking of having the `NexusClient` initializer fire a `PendingDeprecationWarning` warning.
I think that's a lot cleaner and more clearly expresses the true state of the entire Nexus API to any would-be developers.

Especially since these tokens are needed for pretty much any scripting against our APIs at present.

I'll open an issue to discuss.